src/sharding/Shard.js

```diff
@@ -225,6 +228,10 @@ class Shard extends EventEmitter {
    *   .catch(console.error);
    */
   fetchClientValue(prop) {
+    // Shard is dead (maybe respawning), don't cache anything and error immediately
+    if (!this.process && !this.worker) return Promise.reject(new Error('SHARDING_NO_CHILD_EXISTS', this.id));
```

Comment:
This error code does not seem to be defined, it should be defined in the [`Messages.js`](https://github.com/discordjs/discord.js/blob/1e63f3756e814d6b1a2a9c17af9c2b28ce37e472/src/errors/Messages.js#L5) file.

Without that it'd just throw here, which is not the intention:
https://github.com/discordjs/discord.js/blob/1e63f3756e814d6b1a2a9c17af9c2b28ce37e472/src/errors/DJSError.js#L39-L40

Reply:
Ah, didn't know it had to be defined there, some of the other errors that are thrown within Shard also aren't defined there. Will address in a bit 👍